.gitignore

```diff
@@ -12,4 +12,6 @@ com.musala.database.web.parser/build/
 com.musala.database.web.spa.parser/build/
 
 com.musala.database.web.spa.spring.parser/target/
+C:\Users\stefan.simeonov\workspace\com.musala.data.parser\com.musala.xml.parser.executable\src\main\target
```

Comment:
Always ignore a folder relative to the .gitignore file, not a local machine folder. Please correct.